contracts/RoundManager.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity ^0.4.24;
+
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+
+contract RoundManager {
+  using SafeMath for uint;
+
+  // Round number of the last round
+  uint public lastRound;
+  // Block number of the start of the last round
+  // N.B: not necessarily the block at which initializeRound() was called
+  uint public startOfLastRound;
+
+  uint public roundLength;
+  uint public rateLockDealine;
+
+  constructor() {
+    roundLength = 20;
+    rateLockDealine = 5;
+  }
+
+  function timeSinceBeginningOfLastRound() internal view returns (uint) {
+    return block.number.sub(startOfLastRound);
+  }
+
+  modifier onlyBeforeActiveRoundIsLocked() {
```

Comment:
Modifiers should be placed before the constructor.

Reply:
Thanks, didn't know. Who recommended that ? It's not in the ![style guide](https://solidity.readthedocs.io/en/v0.4.24/style-guide.html)

Reply:
It's just a commonly accepted practice by the community.